### PR TITLE
RK-5669 - JVM Azure functions example

### PR DESCRIPTION
**Pull request by orishavit**

### CODE REVIEW

File: java-azure-functions/Readme.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+```
+mvn clean package azure-functions:deploy
+```
+
+3. Set Rookout token either via the Azure Portal or the CLI:
+```
+az functionapp config appsettings set --name jvmazure-20200323170137626 --settings  "ROOKOUT_TOKEN=<token>" --resource-group java-functions-group
+```
+
+4. Run the function
```
Comment (ElDuderinos) [inline java-azure-functions/Readme.md:18]: Is this working consistently? Didn't we see app settings / env vars are inconsistent? Or is this only the JAVA_OPTS and the command line args setting?

Comment (orishavit) [reply java-azure-functions/Readme.md:18]: The env vars are consistent, the JVM command line wasn't

File: java-azure-functions/Readme.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Description
+
+The Rookout SDK is started in the function code using `API.start()`. You can call this function more than once, the SDK will only start one time.
+
+Configuration can be set via environment variables or the RookOptions struct. See https://docs.rookout.com/docs/jvm-setup/.
```
Comment (ElDuderinos) [inline java-azure-functions/Readme.md:24]: Please add instructions on adding the rook dependency

**Comment by BenTheKlein** [comment]
@orishavit What about .Jenkins file? do we want this example to be used in the tests?

**Pull request merged**